CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to RKE #
+
+Thanks for taking the time to contribute to RKE!
+
+### Opening PRs and organizing commits
+PRs should generally address only 1 issue at a time. If you need to fix two bugs, open two separate PRs. This will keep the scope of your pull requests smaller and allow them to be reviewed and merged more quickly.
+
+Each PR needs to be linked to a Github issue in [rancher project](https://github.com/rancher/rancher) which provides as much detail as possible on the change and its motivation. If it is a bug fix, please describe how to reproduce it in the issue. Make the Team or Project Manager aware of the GitHub issue so that they can set the correct milestone. Milestones are very important for the release and QA tracking. Milestones follow the convention: `$RANCHERVERSION-KDM-$MONTH-$YEAR-patches, e.g. `v2.7-KDM-Nov-2023-patches`.
```

Comment:
AFAIK, there are multiple formats of the milestones. I suggest removing the last sentence.